packages/transition-backend/file/definitions/definitions-en.tex

```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
-\hline
-\label{ultra_peak_volume}
-\makecell[r]{Volume d'ultra-pointe (horaire) \\ \textit{Ultra-peak volume (hourly)}} & \[P_{up}\] & \[pass/h\] & \[4 P_{up15}\] & Volume maximum sur 15 minutes extrapolé sur une heure. \\
-\hline
-\label{peak_hour_factor}
-\makecell[r]{Facteur d'heure de pointe \\ \textit{Peak-hour factor (PHF)}} & \[\gamma_{PHF}\] & - & \[\frac{P_{max}}{P_{up}} = \frac{1}{\gamma_{PHC}}\] & Facteur permettant de tenir compte du volume maximum sur 15 minutes. \(0.25 \leq \gamma_{PHF} \leq 1\). \\
-\hline
-\label{peak_hour_coefficient}
-\makecell[r]{Coefficient d'heure de pointe \\ \textit{Peak-hour coefficient (PHC)}} & \[\gamma_{PHC}\] & - & \[\frac{P_{up}}{P_{max}} = \frac{1}{\gamma_{PHF}}\] & Coefficient permettant de tenir compte du volume maximum sur 15 minutes. \(1 \leq \gamma_{PHC} \leq 4\). \\
-\hline
-\label{design_volume}
-\makecell[r]{Volume de conception \\ \textit{Design volume}} & \[P_d\] & \[pass/h\] & \[\gamma_{PHC} P_{max} = P_{up}\] & Volume utilisé pour déterminer les capacités requises sur la ligne ou le coefficient d'utilisation de la capacité offerte. \\
-\hline
-\label{daily_path_volume}
-\makecell[r]{Volume quotidien de parcours \\ \textit{Daily path volume}} & \[{{P_p}_{day}}\] & \[pass/jour\] & \[\sum_{i=0}^{23} {P_i}\] où \(P_i\) représente le volume de passagers entre l'heure \(i\) et l'heure \(i+1\) sur le parcours & Nombre de passagers par jour transportés sur le parcours, dans une direction. \\
-\hline
-\label{daily_line_volume}
-\makecell[r]{Volume quotidien de ligne \\ \textit{Daily line volume}} & \[{{P_L}_{day}}\] & \[pass/jour\] & \[\sum_{i=0}^{23} {P_i}\] où \(P_i\) représente le volume de passagers entre l'heure \(i\) et l'heure \(i+1\) & Nombre de passagers par jour transportés sur tous les voyages de la ligne. \\
-\hline
-\label{potential_demand}
-\makecell[r]{Demande potentielle \\ \textit{Potential demand}} & \[P_{pot}\] & \[pass/h\] & \[P_{lat} + P_{obs}\] & Volume de passagers qui existerait si le service et la tarification étaient optimaux. \\
-\hline
-\label{latent_demand}
-\makecell[r]{Demande latente \\ \textit{Latent demand}} & \[P_{lat}\] & \[pass/h\] & \[P_{pot} - P_{obs}\] & Demande non desservie ou non satisfaite. \\
-\hline
-\label{observed_volume}
-\makecell[r]{Volume observé \\ \textit{Observed volume}} & \[P_{obs}\] & \[pass/h\] & \[P_{pot} - P_{lat}\] & Volume de passagers observé dans le réseau ou la ligne. \\
-\hline
-\end{longtable}
+    R{.25\NetTableWidth}%
+    C{.06\NetTableWidth}%
+    C{.07\NetTableWidth}%
+    C{.21\NetTableWidth}%
+    L{.41\NetTableWidth}%
+  }
+  \hline
+  \makecell[r]{Definition} & \makecell[c]{Symbol} & \makecell[c]{Unit} & \makecell[c]{Expression} & \makecell[l]{Description} \\\hline
```

Comment:
Can you double check this

Reply:
fixed everywhere, needs double \hline after headers